functions/src/services/questionnaireResponse/nicotineScoringService.ts

```diff
@@ -95,6 +102,9 @@ export class NicotineScoringQuestionnaireResponseService extends QuestionnaireRe
       // Store new score
       await this.updateScore(userId, response.id, score)
 
+      // Store FHIR observation
+      await this.storeFHIRObservation(userId, response.id, score)
```

Comment:
Do you now keep the score in both `users/<user-id>/symptomScores` and in `users/<user-id>/<some-score-observation-name>`? Is there a reason to keep both?